linetools/spectra/io.py

```diff
@@ -236,12 +236,24 @@ def readspec(specfil, inflg=None, efil=None, verbose=False, flux_tags=None,
                 tmpsig[gdp] = np.sqrt(1./sig[gdp])
                 sig = tmpsig
                 wave = 10.**wave
-    elif (head0['NAXIS'] == 2) and (head0['NAXIS2'] == 5): # SDSS .fit format
-        fx = hdulist[0].data[0,:].flatten()
-        sig = hdulist[0].data[2,:].flatten()
-        wave = setwave(head0)
+    elif head0['NAXIS'] == 2:
```

Comment:
Good suggestion.  I fear the current header is overly sparse(see below).
When they finally take real data that will change and I'll try to remember
to make this more unique.


SIMPLE  =                    T / conforms to FITS standard                      BITPIX  =                  -32 / array data type                                NAXIS   =                    2 / number of array dimensions                     NAXIS1  =                 2408                                                  NAXIS2  =                  100                                                  EXTEND  =                    T                                                  BRICKNAM= 'elg-100 '                                                            CHANNEL = 'z       '                                                            EXTNAME = 'FLUX    '           / no dimension                                   END